3.BikeSharing360_MultiContainer_App/README.md

```diff
@@ -1,73 +1,58 @@
+installed.
 
-Add the components:
+* Make sure that the ingress controller has been installed (instructions
+[here](https://kubernetes.github.io/ingress-nginx/deploy)).
 
-```
-> kubectl apply -f ComponentSchematic/bikesharing-ui-component.yaml
-> kubectl apply -f ComponentSchematic/bikesharing-email-api-component.yaml
-> kubectl apply -f ComponentSchematic/bikesharing-feedback-api-component.yaml
-> kubectl apply -f ComponentSchematic/bikesharing-profile-api-component.yaml
-```
 
-Validate that the components were created successfully:
+## Quickstart: Deploy the sample through KubeVela
 
-```
-> kubectl get components
+Deploy the application to Kubernetes cluster.
 
-NAME                          AGE
-bikesharing-email-api-v1      11s
-bikesharing-feedback-api-v1   7s
-bikesharing-profile-api-v1    3s
-bikesharing-ui-v1             16s
+```shell
+$ kubectl apply -f ./KubeVelaManifest
 ```
 
-Apply the configuration:
+Validate that the deployments are created successfully by KubeVela.
 
-```
-> kubectl apply -f ApplicationConfiguration/bikesharing-app-configuration.yaml
-```
-
-Validate that the configuration was applied successfully:
+```shell
+$ kubectl get deploy
 
-```
-> kubectl get configurations
-
-NAME              AGE
-bikesharing-app   25s
+NAME                       READY   UP-TO-DATE   AVAILABLE   AGE
+bikesharing-email-api      1/1     1            1           17m
+bikesharing-feedback-api   1/1     1            1           17m
+bikesharing-profile-api    1/1     1            1           17m
+bikesharing-ui             1/1     1            1           17m
 ```
 
 Wait for ingress to be created:
 
-```
-> kubectl get ingress
+```shell
+$ kubectl get ingress
 
-NAME                                     HOSTS             ADDRESS   PORTS   AGE
-bikesharing-ui-trait-ingress             bikesharing.com             80      2s
+NAME             CLASS    HOSTS             ADDRESS     PORTS   AGE
+bikesharing-ui   <none>   bikesharing.com   localhost   80      33s
 ```
 
 Navigating to bikesharing.com after mapping it to the correct IP address (found by `kubectl get services`) should take you to the home page.
 
 ![bikesharing website](./App/img/bikesharing.PNG)
 
-## Blogs posts
-
-Here's links relevant to this project:
-
-- OAM project: [official website](https://oam.dev/)
-- Open Source Blog: [OAM release announcement](https://cloudblogs.microsoft.com/opensource/2019/10/16/announcing-open-application-model/)
-
-## Copyright and license
-* Code and documentation copyright 2016 Microsoft Corp. Code released under the [MIT license](https://opensource.org/licenses/MIT).
-
-## Code of Conduct 
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
```

Comment:
@wonderflow  is it okay to remove this copyright info?

Reply:
why do we need to remove it ?

Reply:
Ok, let me revert it.